index.html

```diff
@@ -15,47 +15,215 @@
+              <li><div class="button2"><button2>Javascript</button2><div><li>
+              <li><div class="button2"><button2>HTML</button2><div><li>    
+            </ul>
+            <div class="tilebut"><button>See Project</button><div>
+        </div>
+      </div>
     </div>
+  </div>
+</div>
 
-  </section>
-  
-  </main>
+<div class="griditem">
+  <div class="placeholder"></div>
+    <div class=whitespace>
+      <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
+          <ul class="minimenu">
+            <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
+            <li><div class="button2"><button2>CSS</button2><div><li>  
+            <li><div class="button2"><button2>Javascript</button2><div><li>
+            <li><div class="button2"><button2>HTML</button2><div><li>    
+          </ul>
+          <div class="tilebut"><button>See Project</button><div>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+</div>
 
-  <footer class="footer">
-  
-  </footer>
+<div class="about">
+  <div>
+    <h1 id="h1p3">About me</h1>
+    <p id="pp3"> I can help you build a product ,<br> feature or website Look through<br> some of my work and experience! <br> If 
+      you like what you see and have a<br> project you need coded,<br>don’t hesitate to contact me.</p>
+      <div><img id="group" src="./Assets/illustration 2-about me mobile.svg" alt="mobile"></div>
+      <button id="pg3but">Get my resume</button>
+  </div>
+</div>
+
+<div class="griditem">
+    <div class=greyspace>
+      <div><img id="language" src="./Assets/icon-languages.svg" alt="Languages"></div>
+      <h3>Languages</h3>
+          <ul class="minimenu">
+            <li class="button3">Ruby on Rails</li>
+            <li class="button3">CSS</li>  
+            <li class="button3">Javascript</li>
+            <li class="button3">HTML</li>    
+          </ul>
+    </div>
+</div>
+
+<div class="griditem">
+  <div class=greyspace>
+    <div><img id="frameworks" src="./Assets/icon-frameworks.svg" alt="Icon"></div>
+    <h3>Frameworks </h3>
+        <ul class="minimenu">
+          <li class="button3">React.js</li>
+          <li class="button3">Ruby on Rails</li>  
+          <li class="button3">RSpec</li><br>      
+        </ul>
+        <ul class="minimenu">
+          <li class="button3">Capybara</li> 
+          <li class="button3">Selenium</li>       
+        </ul>
+  </div>
+</div>
+
+<div class="griditem">
+  <div class=greyspace>
+    <div></div><img id="skills" src="./Assets/icon-skills.svg" alt="Skills"></div>
+    <h3>Skills</h3>
+        <ul class="minimenu">
+          <li class="button3">Database Management</li>
+          <li class="button3">Version Control</li>   
+        </ul>
+
+        <ul class="minimenu"> 
+          <li class="button3">CLI</li>
+          <li class="button3">Web Developement</li>  
+          <li class="button3">API Design</li>  
+        </ul>
+  </div>
+</div>
```

Comment:
- [ ] The technologies should be in the middle of each category. Kindly do the change and try to match the Figma design as well. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign](https://github.com/dvcorreia1984/portfolio/assets/70483835/1954c855-3404-4e5f-90dd-a69c99767b80) | ![figma_about](https://github.com/dvcorreia1984/portfolio/assets/70483835/a0b61c21-69e2-4eec-9d9e-4c80998d3433)